src/core/validator.ts

```diff
@@ -1,83 +1,82 @@
 import chalk from 'chalk';
-import FieldLoader from '../util/fieldLoader';
-import Differ from '../util/differ';
-import GithubFetcher from '../util/githubFetcher';
-import Report from '../util/report';
+import fieldLoader from '../util/fieldLoader';
+import differ from '../util/differ';
+import ghFetcher from '../util/githubFetcher';
+import report from '../util/report';
 
 export interface Options {
   config?: string;
   token?: string;
 }
 
-class Validator {
-  fieldLoader = new FieldLoader();
-  differ = new Differ();
+async function runLocalCheck(options: Options): Promise<void> {
```

Comment:
export const runLocalCheck.. would be more typescripty

Reply:
As I mentioned above arrow functions are also my preference but in this case, because the `runLocalCheck` used internally and not exported it's impossible to mock it when testing. Using normal functions the `rewire` package allows me to mock non exported functions.